receiver/sqlserverreceiver/metadata.yaml

```diff
@@ -255,6 +255,139 @@ metrics:
       input_type: string
     attributes: [database.status]
     extended_documentation: This metric is only available when the receiver is configured to directly connect to SQL Server.
+  sqlserver.active.temp.tables:
+    enabled: false
+    description: The number of active temporary tables.
+    unit: “{temp_tables}”
+    gauge:
```

Comment:
Less important to me but it would probably be more accurate to label this as a non-monotonic, cumulative, sum. Although there are other similar existing metrics that are currently labeled as gauges...

Reply:
I can change in this issue for the newly added ones to non-monotonic.